packages/plugin-trpc/cli/src/lib/generate.ts

```diff
@@ -0,0 +1,15 @@
+import { readFileSync } from 'fs';
+
+import { parsePrismaSchema as prismaParser, PrismaSchema } from '@loancrate/prisma-schema-parser';
+
+export const parsePrismaSchema = () => {
+	return prismaParser(readFileSync('prisma/schema.prisma', { encoding: 'utf8' }));
+};
+
+export const getPrismaModelByName = (name: string, schema: PrismaSchema) => {
```

Comment:
I'm not sure if this is the way we want this in the long term. I took this approach with parsing in the schema as an argument, so I can use the function that returns the entire schema, but also the one that finds the model based on the name, with only having to read the filesystem once.